src/docker/run.go

```diff
@@ -0,0 +1,132 @@
+package docker
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"sync"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/client"
+	"github.com/tarantool/cartridge-cli/src/common"
+)
+
+type RunOpts struct {
+	Name       string
+	ImageTag   string
+	WorkingDir string
+	Cmd        []string
+
+	Volumes map[string]string
+
+	Quiet bool
+	Debug bool
+}
+
+func waitStartOutput(out io.ReadCloser, quiet bool) error {
```

Comment:
I think you should provide an interface that get function and flag and enable spinner if flag is true.

It's not good when you meet waitgroups/channels creations to run spinner over all project.

`run_in_background(myFunction, enableSpinner)`.

Reply:
Yes, I sure that all `spinner` magic should be placed in some common module.

Are you OK with doing it in a separated PR? I've added it to checklist of #242

Reply:
Of course